syntax = "proto3";

service DataService {
  rpc SendData (DataRequest) returns (DataResponse) {}
  rpc SendAudioData (AudioDataRequest) returns (AudioDataResponse) {}
}

message DataRequest {
  string data = 1;
}

message DataResponse {
  string message = 1;
}

message AudioDataRequest {
  bytes data = 1;
}

message AudioDataResponse {
  string message = 1;
}